desk/app/groups.hoon

```diff
@@ -678,7 +679,18 @@
     (state-6-to-7 old)
   =?  cor  !=(~ caz-6-to-7)  (emil caz-6-to-7)
   =?  old  ?=(%7 -.old)  (state-7-to-8 old)
-  ?>  ?=(%8 -.old)
+  ::  v8 -> v9: leave all /contact subscriptions
+  ::
+  =?  cor  ?=(%8 -.old)
+    %+  roll  ~(tap by wex.bowl)
+    |=  [[[=wire =dock] *] =_cor]
+    ?.  ?=([%contact ~] wire)  cor
+    =^  caz=(list card)  subs.cor
+      (~(unsubscribe s [subs bowl]) wire dock)
+    =.  cor  (emil:cor caz)
+    (emit:cor [%pass wire %agent dock %leave ~])
```

Comment:
Isn't this just always `[%pass /contact %agent [our.bowl %contacts] %leave ~]`? That no-ops if the sub isn't there, which is fine. Could be worth simplifying here.